gammapy/astro/darkmatter/profiles.py

```diff
@@ -34,19 +34,22 @@ def scale_to_local_density(self):
         scale = (self.LOCAL_DENSITY / self(self.DISTANCE_GC)).to_value("")
         self.parameters["rho_s"].value *= scale
 
-    def _eval_squared(self, radius, separation):
-        """Squared density at given radius together with the substitution part"""
+    def _eval_substitution(self, radius, separation, squared):
+        """Density at given radius together with the substitution part"""
```

Comment:
It might be important to describe all the parameters in the docstring (and their type).